src/components/TrendingCard/__tests__/TrendingCard.spec.js

```diff
@@ -3,36 +3,63 @@
  * @copyright IBM Security 2020
  */
 
+import classnames from 'classnames';
 import { render } from '@testing-library/react';
 import React from 'react';
 
-import { Link } from '../../..';
-import { Default as TrendingCard } from '../TrendingCard.stories';
+import { Link, TrendingCard } from '../../../';
 
 describe('TrendingCard', () => {
-  it('renders', () => {
-    expect(render(<TrendingCard />).container.firstChild).toMatchSnapshot();
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
+    render(
+      <TrendingCard
+        // Use an empty `href` to avoid misdirected Axe "skip-link" violation:
+        href="#"
+        title="test title"
+        subtitle={<span>test subtitle</span>}
+      />,
+      {
+        // DAP requires a landmark '<main>' in the DOM:
+        container: document.body.appendChild(main),
+      }
+    );
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations('TrendingCard');
   });
 
-  it('renders a custom link', () => {
-    expect(
-      render(<TrendingCard element={Link} />).container.firstChild
-    ).toMatchSnapshot();
-  });
+  test('should accept a custom link', () => {
+    const { getByText, queryByTestId } = render(
+      <TrendingCard
+        title="test title"
+        className="test-trending-card-class"
+        element={({ children, className, ...other }) => (
+          <Link
```

Comment:
If possible, I wonder if here it would also be valuable to test that the `Link` component itself has been rendered.

Reply:
That seems more appropriate for the `Link` component? (What would you check -- the presencee of a class name? that definitely seems more `Link`-test specific). 

There are a couple checks in this block that include `closest('a')` to target the anchor element, coming from the `Link`, as well.